### PR TITLE
0.10.1

### LINKED ISSUE #474: Changelog does not parse correctly for release
## Current behaviour

Changelog used for vagrant box release is blank

## Expected behaviour

Changelog for the current release should parse out of CHANGELOG.md

## How to reproduce?

Try releasing

## Suggestion(s)/solution(s) [Optional]

Make sure it runs always and test that the result is not blank.

## Checklist (after created issue)
- [x] Added label(s)
- [ ] Added to project
- [x] Added to milestone

### LINKED ISSUE #477: Directory has wrong permissions in 0.10
## Current behaviour
after starting vagrant box, 'vagrant ssh' to start an interactive shell, and then as user 'vagrant' try to run any hashicorp command:

```
vagrant@vagrant:~$ consul
-bash: consul: command not found
vagrant@vagrant:~$ vault
-bash: vault: command not found
vagrant@vagrant:~$ nomad
-bash: nomad: command not found
```

On inspecting the state of /usr/local/bin/oss:
```
vagrant@vagrant:~$ ls -al /usr/local/bin/oss
ls: cannot access '/usr/local/bin/oss/consul': Permission denied
ls: cannot access '/usr/local/bin/oss/packer': Permission denied
ls: cannot access '/usr/local/bin/oss/terraform': Permission denied
ls: cannot access '/usr/local/bin/oss/nomad': Permission denied
ls: cannot access '/usr/local/bin/oss/vault': Permission denied
ls: cannot access '/usr/local/bin/oss/consul-template': Permission denied
ls: cannot access '/usr/local/bin/oss/..': Permission denied
ls: cannot access '/usr/local/bin/oss/.': Permission denied
total 0
d????????? ? ? ? ?            ? .
d????????? ? ? ? ?            ? ..
-????????? ? ? ? ?            ? consul
-????????? ? ? ? ?            ? consul-template
-????????? ? ? ? ?            ? nomad
-????????? ? ? ? ?            ? packer
-????????? ? ? ? ?            ? terraform
-????????? ? ? ? ?            ? vault
```
This behaviour is weird but is caused by incorrect permissions on folder /usr/local/bin/oss, which is set to 0744 - eg, no execute for non-owner.

## Expected behaviour
/usr/local/bin/oss has permissions 0755.

## Checklist (after created issue)
- [ ] Added label(s)
- [ ] Added to project
- [ ] Added to milestone

### PR DESCRIPTION
**Pull request by fredrikhgrelland**
## Closes/fixes/resolves issue(s)?

fixes #477 
fixes #474 

## Checklist (after created PR)
- [x] Added reviewers
- [x] Added assignee(s)
- [x] Added label(s)
- [ ] Added to project
- [x] Added to milestone

### CODE REVIEW

**Review by claesgill — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,19 +1,25 @@
 # Changelog
 
+## [0.10.1]
```
Comment (zhenik) [inline CHANGELOG.md:3]: Are we going to make releases manual?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
+
+### Fixed
+- Changelog does not parse correctly for release #474
+- Directory has wrong permissions in 0.10 #477
```
Comment (zhenik) [inline CHANGELOG.md:7]: Why this part was not add together with implementation and in separate PR?

Comment (zhenik) [reply CHANGELOG.md:7]: ok. I see all implemented in this PR

File: CHANGELOG.md (hunk 1 above)
Comment (fredrikhgrelland) [reply CHANGELOG.md:3]: This will become a release automatically - see label

File: CHANGELOG.md (hunk 2 above)
Comment (fredrikhgrelland) [reply CHANGELOG.md:7]: See up

**Pull request merged**